FireRender.Maya.Src/FireMaterialViewRenderer.cpp

```diff
@@ -80,7 +72,7 @@ bool FireMaterialViewRenderer::RunOnFireRenderThread()
 	switch (m_threadCmd)
 	{
 		case ThreadCommand::RENDER_IMAGE: {
-			RPR::AutoMutexLock lock(m_renderData.m_mutex);
+			RPR::AutoMutexLock lock(m_renderDataPtr->m_mutex);
```

Comment:
Please add check if m_renderDataPtr is not empty.
Here and below where such check is absent.

Reply:
Asserts are added